FirebaseTestProject/Models/Post.swift

```diff
@@ -7,24 +7,72 @@
 
 import Foundation
 import FirebaseFirestore
+import SwiftUI
```

Comment:
What do we need `SwiftUI` for in the model here?

Reply:
Deleted